stdlib.go

```diff
@@ -32,6 +32,8 @@ type StdlibAdapter struct {
 	messageKey      string
 	prefix          string
 	joinPrefixToMsg bool
+	expectCaller    bool
+	logRegexp       *regexp.Regexp
```

Comment:
I think it is redundant to have both of these new fields. It seems to me we can add a single bool and use an `if` in the methods to select one of two package level regexps or we can eliminate the bool field and populate the regexp field in the option function. At the moment I would lean toward the second choice because I think it will be more elegant on balance. You might want to try it both ways to see which comes out nicer.

Reply:
I removed the expectCaller. logRegexp is more powerful here.